bundles/org.openhab.core/src/main/java/org/openhab/core/types/util/UnitUtils.java

```diff
@@ -134,7 +129,8 @@ public class UnitUtils {
                                 LOGGER.warn("Unit field points to a null value: {}", field);
                             } else if (systemUnit.equals(unit)) {
                                 return dimension;
-                            } else if (compatibleDimension == null && systemUnit.isCompatible(unit)) {
+                            } else if (compatibleDimension == null
+                                    && (systemUnit.isCompatible(unit) || systemUnit.inverse().isCompatible(unit))) {
```

Comment:
Not 100% sure about this, but wondering if there there a functional difference between `systemUnit.inverse().isCompatible(unit)` and `systemUnit.isCompatible(unit.invers())` ??